### PR TITLE
[Keyboard] Add SyenaKeyboards

### PR DESCRIPTION
**Pull request by syenasweta**
## Description

Add a keyboard.

## Types of Changes

- [x] Core
- [x] Bugfix
- [x] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/syenakeyboards/aswagata/aswagata.c
Comment (waffle87) [inline keyboards/syenakeyboards/aswagata/aswagata.c:1]: This file should be removed.

File: keyboards/syenakeyboards/aswagata/aswagata.h
Comment (waffle87) [inline keyboards/syenakeyboards/aswagata/aswagata.h:1]: This file should be removed.

File: keyboards/syenakeyboards/aswagata/config.h
Comment (waffle87) [inline keyboards/syenakeyboards/aswagata/config.h:1]: This file should be removed.

File: keyboards/syenakeyboards/aswagata/info.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": true,
+        "console": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "lto": true,
+        "key_override": true
```
Comment (waffle87) [inline keyboards/syenakeyboards/aswagata/info.json:15]: * LTO is not enabled like this — belongs under a different key. Not necessary for RP2040 anyway.
* Key overrides cannot be enabled at the keyboard level.
```suggestion
        "nkro": true
```

File: keyboards/syenakeyboards/aswagata/info.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    "processor": "RP2040",
+    "url": "https://github.com/syenasweta/syenakeyboards",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0003",
+        "vid": "0x2E8A"
+    },
+    "usb_detect": {
+        "timeout": 2500
+    }
```
Comment (waffle87) [inline keyboards/syenakeyboards/aswagata/info.json:30]: Not a split keyboard.
```suggestion
```

File: keyboards/syenakeyboards/aswagata/keymaps/via/rules.mk [hunk 3]
```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+SERIAL_DRIVER = vendor
```
Comment (waffle87) [inline keyboards/syenakeyboards/aswagata/keymaps/via/rules.mk:2]:
```suggestion
```

File: keyboards/syenakeyboards/aswagata/readme.md
Comment (waffle87) [inline keyboards/syenakeyboards/aswagata/readme.md:1]: The fields in this file need to be filled out...

File: keyboards/syenakeyboards/aswagata/rules.mk [hunk 4]
```diff
@@ -0,0 +1,2 @@
+# This file intentionally left blank
+SERIAL_DRIVER = vendor
```
Comment (waffle87) [inline keyboards/syenakeyboards/aswagata/rules.mk:2]:
```suggestion
```

File: keyboards/syenakeyboards/readme.md
Comment (waffle87) [inline keyboards/syenakeyboards/readme.md:1]: This file should be removed.

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/syenakeyboards/readme.md
Comment (syenasweta) [reply keyboards/syenakeyboards/readme.md:1]: Yes, I've remove.

File: keyboards/syenakeyboards/aswagata/rules.mk (hunk 4 above)
Comment (syenasweta) [reply keyboards/syenakeyboards/aswagata/rules.mk]: I've remove it.

File: keyboards/syenakeyboards/aswagata/readme.md
Comment (syenasweta) [reply keyboards/syenakeyboards/aswagata/readme.md:1]: I have completed the fields in this file.

**Review by syenasweta — COMMENTED** [review]
I've reviewed and edited it

File: keyboards/syenakeyboards/aswagata/config.h
[the remark this answers is not in the document]
Comment (syenasweta) [reply keyboards/syenakeyboards/aswagata/config.h:1]: Ok, it's been deleted.

File: keyboards/syenakeyboards/aswagata/readme.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# syenakeyboards/aswagata
+
+![SyenaKeyboards Aswagata](https://images.tokopedia.net/img/cache/900/VqbcmM/2023/9/30/dc3e20b5-eb94-4f92-8a44-f2f556dcc1ab.jpg)
```
[the remark this answers is not in the document]
Comment (syenasweta) [reply keyboards/syenakeyboards/aswagata/readme.md]: I've uploaded it on Imgur and changed the `url` of the image.

File: keyboards/syenakeyboards/aswagata/readme.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+Aswagata as a macropad has four buttons with default functions: `copy`, `paste`, `cut`, and `select` (for Linux). It can be reconfigured with Via.
```
[the remark this answers is not in the document]
Comment (syenasweta) [reply keyboards/syenakeyboards/aswagata/readme.md]: Thanks for the correction.

File: keyboards/syenakeyboards/aswagata/readme.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+    make syenakeyboards/aswagata:default:flash
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+## Bootloader
+
+Enter the bootloader in 3 ways:
+
+* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Copy key) and plug in the keyboard
```
[the remark this answers is not in the document]
Comment (syenasweta) [reply keyboards/syenakeyboards/aswagata/readme.md]: Again, thanks for suggesting.

File: keyboards/syenakeyboards/aswagata/readme.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
```
[the remark this answers is not in the document]
Comment (syenasweta) [reply keyboards/syenakeyboards/aswagata/readme.md]: Oh yes. I missed reading the `readme.md` template. I've deleted it.

**Review by syenasweta — COMMENTED** [review]
I've updated `readme.md` and deleted `config.h`. Thanks for the review.

File: keyboards/syenakeyboards/aswagata/readme.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
+# syenakeyboards/aswagata
+
+![SyenaKeyboards Aswagata](https://i.imgur.com/YIcg9Cg.jpg)
```
[the remark this answers is not in the document]
Comment (syenasweta) [reply keyboards/syenakeyboards/aswagata/readme.md]: Hi, I've replaced it.

**Review by syenasweta — COMMENTED** [review]
Update readme .md for syenakeyboards/aswagata

**Review by drashna — APPROVED** [review]

**Review by waffle87 — APPROVED** [review]

**Pull request merged**